src/meta/src/hummock/compaction/picker/tier_compaction_picker.rs

```diff
@@ -302,9 +304,11 @@ impl TierCompactionPicker {
                 table_infos: level.table_infos.clone(),
             }];
 
+            // We assume that the maximum size of each sub_level is sub_level_max_compaction_bytes,
+            // so the design here wants to merge multiple overlapping-levels in one compaction
             let max_compaction_bytes = std::cmp::min(
                 self.config.max_compaction_bytes,
-                self.config.sub_level_max_compaction_bytes,
+                self.config.sub_level_max_compaction_bytes * 4,
```

Comment:
should replace `4` with `vnode_split_cnt`?

Reply:
What's the meaning of `4` here? May use a constant instead of a magic number.